core/embed/rust/src/ui/model_tt/component/page.rs

```diff
@@ -58,14 +81,26 @@ where
         // paint.
         self.fade = Some(theme::BACKLIGHT_NORMAL);
     }
+
+    fn paint_hint(&mut self) {
+        display::text_center(
+            Point::new(self.pad.area.center().x, self.pad.area.bottom_right().y - 3),
```

Comment:
I don't remember now if we have a `Rect::bottom_center()` so the params could use the same source point, but if not, maybe we should!

Reply:
fixed in e0abc6c.